keel-core/src/main/kotlin/com/netflix/spinnaker/keel/constraints/ConstraintEvaluator.kt

```diff
@@ -6,6 +6,24 @@ import com.netflix.spinnaker.keel.api.DeliveryConfig
 
 interface ConstraintEvaluator<T : Constraint> {
 
+  companion object {
+    inline fun <reified T> getConstraintForEnvironment(
```

Comment:
Kotlin newbie question: does this need to be `reified` if you're not checking/using the type of T in the method?

Reply:
Correct. Good spot.

Reply:
Kotlin studies paying off! 😄